### PR TITLE
Documents are contradictory for Update scan_acr.adoc

### PR DESCRIPTION
**Pull request by timberbee**
The current version says to use credentials from the pull down menu, however you cannot you Prisma Cloud credentials to authenticate to an Azure Container Registry. This is explained in the authentication section but if you come straight to how to configure your registry you will not get that information. Instead you will get frustrated.

### CODE REVIEW

**Comment by Pubs-MV** [comment]
please review this for 22.12.

Also, let Ian know if the same changes are applicable for 22.06.

File: compute/admin_guide/vulnerability_management/registry_scanning/scan_acr.adoc [hunk 1]
```diff
@@ -34,7 +34,7 @@ Leave this field blank to scan all images, regardless of their tag.
 
 .. In the *Credential* field, configure how Prisma Cloud authenticates with ACR.
 +
-Select a credential from the drop-down list.
+You need to create a Service Principal with Reader Role and scoped for the subscription in order to scan the registry. You cannot use the Cloud Accounts added to Prisma Cloud even though they have the correct permissions. For more information see [credentials_store](/authentication/credentials_store)
```
Comment (rcaballeromx) [inline compute/admin_guide/vulnerability_management/registry_scanning/scan_acr.adoc:37]: Thank you @timberbee I've edited the text and fixed the link's syntax. 
```suggestion
To scan the registry, you need a service principal in Azure with the `Reader` role and scoped for your subscription. The Azure account you added to Prisma Cloud can't perform the scans even if they have the correct permissions. For more information, see xref:../authentication/credentials-store[credentials store]
```

**Review by rcaballeromx — CHANGES_REQUESTED** [review]
Please accept the suggestion to apply the needed changes.

**Comment by rcaballeromx** [comment]
@Pubs-MV I think this also applies to 22.06. @timberbee Can you confirm?

**Comment by Pubs-MV** [comment]
@rcaballeromx can you please review and confirm if these changes are required/worked into our content for lagrange so we can  close this PR with the link to what is done.

**Pull request closed without merging**